common/app/model/meta.scala

```diff
@@ -548,6 +548,7 @@ final case class Tags(
   lazy val types: List[Tag] = tagsOfType("Type")
   lazy val tracking: List[Tag] = tagsOfType("Tracking")
   lazy val paidContent: List[Tag] = tagsOfType("PaidContent")
+  lazy val isHosted: Boolean = tagsOfTypeOrPaidContentSubtype("PaidContent", "HostedContent").nonEmpty
```

Comment:
I think this will be true for all paid content, whether hosted or not:
https://github.com/guardian/frontend/blob/a0d9ef12b02c50962c6c2e48431248ff30547575/common/app/model/meta.scala#L534